osu.Game/OsuGame.cs

```diff
@@ -609,6 +609,9 @@ protected override void LoadComplete()
 
             loadComponentSingleFile(screenshotManager, Add);
 
+            // dependency on notification overlay, dependent by settings overlay
+            loadComponentSingleFile(CreateUpdateManager(), Add, true);
```

Comment:
are you sure this doesn't regress anything?

Reply:
![image](https://user-images.githubusercontent.com/22963120/81357283-87427480-90d3-11ea-819e-b0d980ca27b5.png)
osu!'s update check at startup still works, so this is a yes from me.